jdl/parsing/jdl-ast-builder-visitor.ts

```diff
@@ -352,7 +357,7 @@ module.exports = class JDLAstBuilderVisitor extends BaseJDLCSTVisitor {
   }
 
   enumProp(context) {
-    const prop = {
+    const prop: any = {
```

Comment:
I think we need a new issue for handling the files related to chevrotain and the parsing system

Reply:
Because doing it in this PR would take a long time

Reply:
I created an issue for that: https://github.com/jhipster/generator-jhipster/issues/20035